distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Format.enso

```diff
@@ -69,4 +72,11 @@ type Excel_Format
         case self.section of
             Excel_Section.Sheet_Names -> Error.throw (Illegal_Argument.Error "Sheet_Names cannot be used for `write`.")
             Excel_Section.Range_Names -> Error.throw (Illegal_Argument.Error "Range_Names cannot be used for `write`.")
+            Excel_Section.Workbook ->
+                sheet_name = if file.exists.not then "Sheet1" else
+                    names = Excel_Reader.read_file file Excel_Section.Sheet_Names False on_problems format
+                    name_map = Map.from_vector (names.map n-> [n, 1])
+                    idx = 1.up_to 256 . find n-> (name_map.contains_key ("Sheet" + n.to_text) . not)
+                    "Sheet" + idx.to_text
+                Excel_Writer.write_file file table on_existing_file (Excel_Section.Worksheet sheet_name) True match_columns on_problems format
```

Comment:
I think I'd like to see this documented somewhere and currently I did not find any documentation.

This logic is not something I'd know intuitively, so as a user trying to write to an Excel file I'd expect to see somewhere a piece of documentation telling me what is supposed to happen and how can I control it.

For example, one thing that is a bit surprising is that subsequent writes of the same table will create new sheets. Makes sense, but I could expect them to overwrite the same one.

Reply:
I think maybe we should add this documentation to the `Excel` format type?